sql-plugin/src/main/scala/org/apache/spark/sql/rapids/collectionOperations.scala

```diff
@@ -108,6 +108,122 @@ case class GpuMapConcat(children: Seq[Expression]) extends GpuComplexTypeMerging
     }
 }
 
+case class GpuArrayJoin(override val children : Seq[Expression])
+  extends GpuExpression with ShimExpression {
+
+  private val array = children(0)
```

Comment:
Nit: Typically do not use universal keyword as variable name. Instead of `array` we can use `input` or `array_data` or something similar.